bindings/go/Makefile.am

```diff
@@ -37,14 +37,19 @@ bench: $(GOTMPDIR)
 	$(GO) env -w GO111MODULE=off ; \
 	LD_LIBRARY_PATH=$(UCX_SOPATH):${LD_LIBRARY_PATH} $(GO) test -v -bench=.
 
-build-perftest: $(GOTMPDIR)
+goperftest: $(GOTMPDIR)
 	cd $(abs_top_srcdir)/bindings/go/src/examples/perftest ;\
 	$(GO) build -o ${GOTMPDIR}/goperftest
 
 run-perftest:
 	cd $(abs_top_srcdir)/bindings/go/src/examples/perftest ;\
 	LD_LIBRARY_PATH=$(UCX_SOPATH):${LD_LIBRARY_PATH} ${GOTMPDIR}/goperftest ${ARGS}
 
-all: build-perftest build
+install-exec-hook:
+	cp ${GOTMPDIR}/goperftest $(DESTDIR)$(bindir)
```

Comment:
minor: it's more customary to use `$(INSTALL)` command (though is basically same as cp)